pkg/webui/console/components/user-data-form/index.js

```diff
@@ -149,7 +166,7 @@ class UserForm extends React.Component {
           title={sharedMessages.userId}
           name="ids.user_id"
           component={Input}
-          disabled
+          disabled={!create}
```

Comment:
```suggestion
          disabled={!create}
          autofocus={create}
```

> Auto focus the first input when:
> 
> *   The form is a create form

https://github.com/TheThingsNetwork/lorawan-stack/blob/develop/DESIGN.md#forms